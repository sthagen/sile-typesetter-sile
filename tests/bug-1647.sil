\begin[papersize=a7]{document}
\use[module=packages.retrograde,target=v0.15.0]
\neverindent
\nofolios
\set[parameter=document.parskip, value=50pt]
\par% bug 1647 was caused to the parskip introduced here, skipped at outputting
% but counted when adjusting vertical glues.
This paragraph is at the page top.

This paragraph has a big parskip before it.
\vfill
This paragraph must be at the page bottom.
\par
\break
\end{document}
