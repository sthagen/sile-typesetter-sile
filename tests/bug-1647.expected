Set paper size 	209.7637818	297.6377985
Begin page
Mx 	10.4882
My 	22.4502
Set font 	Gentium Plus;10;400;;normal;;;LTR;
T	55 75 76 86 w=17.5977	(This)
Mx 	30.7338
T	83 68 85 68 74 85 68 83 75 w=42.6807	(paragraph)
Mx 	76.0625
T	76 86 w=6.5723	(is)
Mx 	85.2827
T	68 87 w=8.0322	(at)
Mx 	95.9629
T	87 75 72 w=13.5791	(the)
Mx 	112.1900
T	83 68 74 72 w=19.3848	(page)
Mx 	134.2227
T	87 82 83 w=13.7695	(top)
Mx 	147.9922
T	17 w=2.2900	(.)
Mx 	10.4882
My 	84.4502
T	55 75 76 86 w=17.5977	(This)
Mx 	30.7299
T	83 68 85 68 74 85 68 83 75 w=42.6807	(paragraph)
Mx 	76.0546
T	75 68 86 w=13.9697	(has)
Mx 	92.6684
T	68 w=4.5898	(a)
Mx 	99.9023
T	69 76 74 w=12.6660	(big)
Mx 	115.2123
T	83 68 85 86 78 76 83 w=30.5957	(parskip)
Mx 	148.4521
T	69 72 73 82 85 72 w=26.4160	(before)
Mx 	177.5121
T	76 87 w=6.1523	(it)
Mx 	183.6645
T	17 w=2.2900	(.)
Mx 	10.4882
My 	265.4326
T	55 75 76 86 w=17.5977	(This)
Mx 	30.7291
T	83 68 85 68 74 85 68 83 75 w=42.6807	(paragraph)
Mx 	76.0531
T	80 88 86 87 w=20.6348	(must)
Mx 	99.3311
T	69 72 w=9.6973	(be)
Mx 	111.6716
T	68 87 w=8.0322	(at)
Mx 	122.3471
T	87 75 72 w=13.5791	(the)
Mx 	138.5695
T	83 68 74 72 w=19.3848	(page)
Mx 	160.5976
T	69 82 87 87 82 80 w=30.0537	(bottom)
Mx 	190.6513
T	17 w=2.2900	(.)
New page
End page
Finish
